dist/package.json

```diff
@@ -0,0 +1,12 @@
+{
+	"name": "cbioportal-frontend",
+	"version": "0.0.3",
+	"description": "",
+	"author": "",
+	"license": "",
+	"pre-commit": [
+		"test",
```

Comment:
You can always run these things yourself with `npm run test`, `npm run build` and then run `git commit -n` to ignore the precommit hook.